### ISSUE TITLE
default_theme not respected while guest access

### ISSUE THREAD

**Issue by paboum** [vector-im/element-web#23893]
### Steps to reproduce

```json
    "default_theme": "dark",
```
1. Open my instance page
2. Click "Explore rooms"
3. Join any room

### Outcome

I can switch the theme with the gear icon in the lower left corner but the default is light despite the config.

### Will you send logs?

No

**Comment by t3chguy** [comment]
> default_theme not respected while guest access
> Click "Explore rooms"
Join any room

I'm confused, guests aren't permitted access to join a room, they're forced to log in or register, so maybe proving a video of the interaction might help.

**Comment by paboum** [comment]
Okay, let me rephrase:
1. Open any instance with `allow_guest_access: true` in their homeserver.yaml.
2. Click "Explore rooms"
3. **Preview** any room which has "Enable guest access" option enabled.
